src/components/Pool/DimBlurBgBox/index.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components'
+import { Box } from 'rebass'
+import { gradients } from '../../../utils/theme'
+
+export const DimBlurBgBox = styled(Box)`
+  border-radius: 12px;
+  border: 1px solid ${({ theme }) => theme.purple5};
+
+  background: ${gradients.purpleDimDark};
+  background-blend-mode: normal, overlay, normal;
+  backdrop-filter: blur(25px);
+`
```

Comment:
we are using it across multiple components in `/Pool` folder, only one time outside in `pages/pools/Pair`, I think it makes sense to be on `components/pool`.

I can renamed it to styleds.tsx because it's only a styled component but it's other thing that's is not consistent in our codebase.